userguide/tutorials/getting_started.adoc

```diff
@@ -1,40 +1,78 @@
 = Getting Started
 
+This document explains what you need to get started using KillBill in your chosen environment. The first step is installing KillBill itself and the Kaui administrative console, along with their required database, in the environment you will be using.
+
 == Installation
 
-=== AWS
+There are several different ways to get KillBill and Kaui up and running. The packages can be installed on your local computer (Windows, Mac, or Linux) or on a Linux server in the cloud. The three principal options are:
+
+1. Use our one-click installer to install the programs using a ready-to-run container on a fresh AWS Ubuntu instance that you provide
+2. Install Docker on your local machine or a new or existing cloud server instance. Then install KillBill and Kaui in a Docker container.
+3. Install KillBill and Kaui, together with a MySQL database, directly on an Apache Tomcat server in your chosen environment.
+
+These options are discussed in the remainder of this document.
 
-Already familiar with AWS? Running Kill Bill on AWS using our http://docs.killbill.io/latest/aws.html[One-Click installer] is the easiest and fastest way to get started.
+=== AWS (One-Click)
+
+If you prefer to run KillBill on a cloud server, and are familiar with AWS, this is the easiest and fastest way to get started. Just try out our http://docs.killbill.io/latest/aws.html[One-Click installer]. 
 
 === Docker
 
-You can run Kill Bill locally using our official Docker images.
+You can run Kill Bill locally or in the cloud using our official Docker images. These require the installation of Docker and Docker Compose. Docker Compose is used to setup and manage the three separate Docker containers required by KillBill.
```

Comment:
Maybe restate what those are. Something like:

, namely `killbill` server, `kaui` (admin UI) and the SQL engine

Reply:
I added a statement for this